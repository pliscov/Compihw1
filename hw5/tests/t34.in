void main()
{
    print("Checking ternary order of operations");
    int x = 0 if (7>8) else 1 if (10>9) else 2 if (6!=6) else 3 if (6==6) else 4;
    int y = (0 if (7>8) else (1 if (10>9) else (2 if (6!=6) else (3 if (6==6) else 4))));

    if (x == y)
    {
        print("PASSED");
        return;
    }
    print("FAILED");
}