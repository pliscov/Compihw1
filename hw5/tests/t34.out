Checking ternary order of operations
PASSED
